common/changes/leddie24-breadcrumb-focus-rect_2017-01-20-21-59.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Breadcrumb: Fix focus rect",
```

Comment:
It would be great if you could be more descriptive in describing how the focus rect is fixed.